zipkin-server/src/main/resources/banner.txt

```diff
@@ -1,25 +1,28 @@
-                                    ********
-                                  **        **
-                                 *            *
-                                **            **
-                                **            **
-                                 **          **
-                                  **        **
-                                    ********
-                                      ****
-                                      ****
-        ****                          ****
-     ******                           ****                                 ***
-  ****************************************************************************
-    *******                           ****                                 ***
-        ****                          ****
-                                       **
-                                       **
-
-
-             *****      **     *****     ** **       **     **   **
-               **       **     **  *     ***         **     **** **
-              **        **     *****     ****        **     **  ***
-             ******     **     **        **  **      **     **   **
+${AnsiColor.RED}
+                           o
```

Comment:
nit: I feel is missing one dot on the left

```suggestion
                          oo
```

Reply:
Fixed :)